code/style.css

```diff
@@ -1,7 +1,232 @@
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
+body{
+  font-family: 'montserrat', 'helvetica Neue', 'sans-serif';
+}
+
+/* Header/Hero section with video and text */
+.hero{
+  height: 25vh;
+  width: 100vw;
+  position: relative;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.hero video{
+  position: absolute;
+  left: 0;
+  top: 0;
+  object-fit: cover;
+  height: 100%;
+  width: 100%;
+}
+
+.hero-h1{
+  position: absolute;
+  color: white;
+  text-shadow: -1px -1px 2px black;
+  font-size: 400%;
+}
+
+/* text section */
+.text-wrapper{
+  display: flex;
+  flex-direction: column;
+  height: 20vh;
+  margin: 0.5rem;
+  padding: 0.5rem;
+  justify-content: center;
+  text-align: center;
+  font-size: 10px;
+}
+
+h2{
+  margin: 0.5rem;
+  color: #a96e52;
+}
+
+/* signup form and accordion section*/
+
+@media (max-width: 767px;){
+  .content-wrapper{
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+} 
+
+.form-wrapper{
+  height: 50vh;
+  min-height: 50vh;
+  width: 100vw;
+  margin: 0.5rem;
+  background-image: url(./img/jeremy-bezanger-QJ7gHF6V4no-unsplash.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  display: grid;
+  justify-items: center;
+  align-items: center;
+}
+  
+form{
+  background-color: white;
+  color: #a16454;
+  border-radius: 2rem;
+  padding: 2rem;
+  height: 80%;
+  display: grid;
+}
+  
+h1{
+  color: #ffc5c4;
+}
+  
+.accept{
+  font-size: 12px;
+}
+  
+button{
+  width: 100px;
+  border-radius: 2rem;
+  border: none;
+  background-color: #ffc5c4;
+  padding: 0.5rem;
+  margin: 1rem;
+  font-size: 1rem;
+  cursor: pointer;
+}
+
+button:hover{
+  color: palevioletred;
+}
+
+.accordion-wrapper{
+  margin: 0.5rem;
+  display: grid;
+  justify-items: center;
+  background-color: #fbb98a;
+  width: 100vw;
+  height: 50vh;  
+}
+  
+.faq{
+  color: #a16454;
+  align-self: flex-end;
+  font-size: 9rem;
+}
+    
+.question{
+  width: 50vw;
+  height: 10vw;
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  border-radius: 0.5rem;
+  background: white;
+  font-size: 12px;
+  margin-top: 1rem;
+  color: #a16454;
+}
+
+.question:hover{
+  color: #ffc5c4 ;
+}
+
+.answer{
+  background-color: #ffc5c4;
+  color: #a96e52;
+  display: none;
+  width: 50vw;
+  height: 10vw;
+  border-radius: 0.5rem;
+  text-align: center;
+  border: 5px black;
+  font-size: 12px;
+}
+  
+
+/* section for styling of accordion */
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
 .active + .answer {
```

Comment:
```.active + .answer { display: block;  }``` In the desktop version of the page, the accordion content stretches outside the content edges of its container. Try to add overflow: hidden; and see if that solves the problem.

Reply:
Added oveflow: hidden; but still have the same problem.  Will se if i can find another solution.